components/k8s-model-server/iap-client/REAMDE.md

```diff
@@ -0,0 +1,60 @@
+# Serving with IAP enabled
+
+## Setup
+
+Follow this [doc](https://github.com/kubeflow/kubeflow/blob/master/docs/gke/iap.md) to
+setup the cluster with IAP enabled. Save the client id as `IAP_CLIENT_ID`
+
+Deploy a [TF serving component](https://github.com/kubeflow/kubeflow/tree/master/components/k8s-model-server)
+with a mnist model.
```

Comment:
Can we generalize the instructions so they aren't model specific?

Reply:
It's not model specific, I was just using it as an example. Removed it.